src/main/java/pl/isa/biblioteka/user/Users.java

```diff
@@ -0,0 +1,80 @@
+package pl.isa.biblioteka.user;
+
+import pl.isa.biblioteka.books.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.function.Predicate;
+
+public class Users {
+    public static Person person;
+    public static List<Person> users = new ArrayList<>(PersonService.readUsers());
+    Scanner scanner = new Scanner(System.in);
+
+    public void addUser() { // dodanie loginu i hasła oraz sprawdzenie czy istnieje już użytkownik o wskazanym loginie.
+        List<Person> persons = Users.getUsers();
+        Scanner input = new Scanner(System.in);
+        System.out.println("Podaj imię nowego użytkownika: ");
+        String firstName = input.nextLine().trim();
+        System.out.println("Podaj nazwisko nowego użytkownika: ");
+        String lastName = input.nextLine().trim();
+        System.out.println("Podaj login nowego użytkownika: ");
+        String login = input.nextLine().trim();
+        System.out.println("Podaj hasło nowego użytkownika: ");
+        String password = input.nextLine().trim();
+        boolean found = false;
+        for (Person person1 : persons) {
+            if (person1.getLogin().equalsIgnoreCase(login)) {
+                found = true;
+            }
+        }
+        if (found) {
+            System.out.println("Użytkownik: " + login + " istnieje w naszej bazie, wprowadź inny login\n");
+        } else {
+            int id = 1;
+            if (!users.isEmpty()) {
+                Person lastPerson = users.get(users.size() - 1);
+                id = lastPerson.getId() + 1;
+            }
+            Person addPerson = new Person(firstName, lastName, id, login, password);
+            users.add(addPerson);
+            System.out.printf("Dodano nowego użytkownika: %d%n %s %s ", id, firstName, lastName);
+        }
+    }
+
+    public boolean deleteUser() {  // metoda Kingi delikatnie przerobiona podaję ID zamiast last name
```

Comment:
Takie komentarze są zbędne, ewentualnie jak są istotne, to nad metodą, nie po nawiasie

Reply:
usunięte